main.cpp

```diff
@@ -168,15 +207,30 @@ struct DAW
         void convertMidi();
     };
     
-    //master music
-    void masterMusic(bool masteredMusic = true, bool error = false);
-    //process audio
-    int processAudio(int NumberOfTracks = 24);
-    //mix music
-    float mixMusic(float audioLevel = 0.9f);
+    
+    void masterMusic(bool masteredMusic = true);
+    void processAudio(int NumberOfTracks = 24);
+    void mixMusic(float audioLevel = 0.9f);
 
 };
 
+void DAW::masterMusic(bool masteredMusic)
+{
+    if(numberofInputs == 2)
+    masteredMusic = true;
```

Comment:
you need to indent the code that will be executed if the `if` expression is `true`
```
if( some expression is true )
    do this thing next
```
- [ ] revise